fix(css): add color space info to gradient pages (#30047)
mdn/content: files/en-us/web/css/gradient/linear-gradient/index.md

{{EmbedLiveSample("Gradient_that_starts_at_60_of_the_gradient_line", 120, 120)}}

### Interpolation in rectangular color space

```css hidden
body {
  width: 100vw;
  height: 100vh;
}
```

```css
body {
  background: linear-gradient(90deg in oklab, blue, red);
}
```

{{EmbedLiveSample("Interpolation in rectangular color space", 120, 120)}}

> ### Interpolation with hue interpolation methods

Reviewer: same edit

```html hidden
<div class="shorter">shorter hue</div>
<div class="longer">longer hue</div>
```

```css hidden
div {
  height: 50vh;
  color: white;
  font-weight: bolder;
}
```

> In this example for interpolation hsl color system is being used and hue is being interpolated.

Reviewer: ```suggestion
In this example, the hsl color system is being used and hue is being interpolated.
```
link hsl and hue

Author: done

> In this example for interpolation [hsl](/en-US/docs/Web/CSS/color_value/hsl) color system is being used and [hue](/en-US/docs/Web/CSS/hue) is being interpolated.

Reviewer: ```suggestion
In this example for interpolation, [hsl](/en-US/docs/Web/CSS/color_value/hsl) color system is being used and [hue](/en-US/docs/Web/CSS/hue) is being interpolated.
```

```css
.shorter {
  background: linear-gradient(90deg in hsl shorter hue, red, blue);
}

.longer {
  background: linear-gradient(90deg in hsl longer hue, red, blue);
}
```

> The box on the top uses [shorter interpolation](/en-US/docs/Web/CSS/hue-interpolation-method#shorter) so color goes straight from red to blue taking short arc on [color wheel](/en-US/docs/Glossary/Color_wheel). And the box on the bottom uses [longer interpolation](/en-US/docs/Web/CSS/hue-interpolation-method#longer) so the color goes from red to blue via green taking longer arc.

Reviewer: same edits as the example above. Do you want to use different colors so we have two slightly different short and long examples?

Author: Updated all the docs with above review comments. Examples with different colors would've made sense if the examples were on one page in same section. On separate pages readers won't even notice.
